reflex/components/forms/checkbox.py

```diff
@@ -48,14 +49,15 @@ class Checkbox(ChakraComponent):
     # The spacing between the checkbox and its label text (0.5rem)
     spacing: Var[str]
 
-    def get_controlled_triggers(self) -> Dict[str, Var]:
+    def get_event_triggers(self) -> dict[str, Union[Var, Any]]:
```

Comment:
We could define a type `Callable[Var, List[Var]]` for the lambda type and use that instead of `Any`. Also, when would `Var` be the dictionary value type, wouldn't it always be a Callable?

Reply:
I put Any because my previous definition using LambdaTypes was causing a lot of pyright errors.

Also the Union[Var, ...] is for backward compatibility until removal of deprecated features.